pages/githubissues.md

```diff
@@ -10,6 +10,8 @@ After this, you will then research the possible solutions or ideas on how to add
 
 Every time you start to work on an issue you need to create a branch to keep your work on various issues separate from each other. Before you create your branch, sync your repo following the process in the [previous step](http://open-learning-exchange.github.io/#!pages/gitandrepositories.md). Then use  `git checkout -b name_of_your_new_branch --track origin/name_of_your_new_branch` to create and switch to a new branch.
 
+If the above command gives you an error, try this as an alternative. Use `git checkout -b new-branch` to create a new branch. Then create an initial commit `git commit -m "commit message"` and finally push the the newly created branch using `git push -u origin new-branch`.
+
```

Comment:
Sorry, my bad, I should have explained better.
Let's try to do a nice job on this. Look at issue #379. If you use that method to create a branch on GitHub, then you should use `git checkout -b name_of_your_new_branch --track origin/name_of_your_new_branch`, otherwise you should use the method I suggested.